dependencies.yaml

```diff
@@ -0,0 +1,178 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: conda
+    matrix:
+      cuda: ["11.5"]
+      arch: [x86_64]
+    includes:
+      - build
+      - develop
+      - run
+      - test
+      - doc
+      - notebook
+  test:
+    output: none
+    includes:
+      - test
+channels:
+  - rapidsai
+  - conda-forge
+# Not sure whether we need these:
+# - rapidsai-nightly
+# - dask/label/dev
+# - nvidia
+dependencies:
+  build:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - cmake>=3.23.1
+          - cython>=0.29,<0.30
+          - dlpack>=0.5,<0.6.0a0
+          - pyarrow=9.0.0
+          - rmm=22.12.*
+          - scikit-build>=0.13.1
+      - output_types: conda
+        packages:
+          - arrow-cpp=9
+          - c-compiler
+          - cxx-compiler
+          - librdkafka=1.7.0
+          - protobuf>=3.20.1,<3.21.0a0
+          - python>=3.8,<3.10
+    specific:
+      - output_types: [conda, requirements]
+        matrices:
+        - matrix:
+            cuda: "11.5"
+          packages:
+            - cuda-python>=11.7.1,<12.0
+      - output_types: conda
+        matrices:
+        - matrix:
+            arch: x86_64
+          packages:
+            - gcc_linux-64=9.*
+            - sysroot_linux-64==2.17
+        - matrix:
+            arch: aarch64
```

Comment:
ToDo: check `arch` string

Reply:
Checked. The string is `aarch64` as expected.

Reply:
Confirmed [here](https://github.com/rapidsai/rmm/actions/runs/3395618780/jobs/5681030561#step:6:94)